src/main/resources/application.properties

```diff
@@ -1 +1,15 @@
 spring.application.name=shopyy
+#Database
+spring.datasource.url=jdbc:mysql://localhost:3306/security1
+spring.datasource.username=root
+spring.datasource.password=hoangan1109
```

Comment:
after doing it locally, change this password to root again before committing

Reply:
resolve